shub/image/list.py

```diff
@@ -140,12 +143,17 @@ def raise_shub_image_info_error(error):
     if not isinstance(spiders_list, list):
         raise_shub_image_info_error('spiders section must be a list')
 
+    project_type = metadata.get('project_type')
     spiders, scripts = [], []
     for name in spiders_list:
         if not (name and isinstance(name, string_types)):
             raise_shub_image_info_error("spider name can't be empty or non-string")
-        scripts.append(name[3:]) if name.startswith('py:') else spiders.append(name)
+        if project_type == 'scrapy' and name.startswith('py:'):
```

Comment:
My guess is that scripts are something scrapy-specific, so it looks valid to me to add an additional condition here.